foursight_core/deploy.py

```diff
@@ -129,14 +161,14 @@ def build_config(cls, stage, identity=None, stack_name=None,
                     if (global_bucket_env_from_environ
                             and global_env_bucket_from_environ
                             and global_bucket_env_from_environ != global_env_bucket_from_environ):
-                        print('ERROR. GLOBAL_BUCKET_ENV and GLOBAL_ENV_BUCKET are both set, but inconsistently.')
+                        PRINT('ERROR. GLOBAL_BUCKET_ENV and GLOBAL_ENV_BUCKET are both set, but inconsistently.')
                         sys.exit()
```

Comment:
Seems like this should be `sys.exit(1)`

Reply:
(Also is `sys.exit` different than `exit` here? `exit` is a built-in.)

Reply:
Sounds good, sys.exit(1). And: https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python ... "exit is a helper for the interactive shell - sys.exit is intended for use in programs."